lib/index.js

```diff
@@ -1167,22 +1167,38 @@ module.exports = (function() {
             }
             return ab;
         };
-        substitution = self.imageToBuffer(substitution)
+        try {
+            substitution = self.imageToBuffer(substitution)
+        } catch (error) {
```

Comment:
I do not think that we should hide nderlying error from the develop.
He should understand that `substitution` is broken in specific way and fix that in his code.

Reply:
If I want, in some case, to continue the substitution even if image error (i.e not exist).
Wich algo can I make ? Add an option like `catchImageError` ?

Reply:
I think function `handleImageError(subsitution): Buffer | null` will do the trick. If result is `null` (unhandled) then we rethrow error using `throw`. That will require additional test.

Reply:
And this function `handleImageError(subsitution): Buffer | null` will be override by the develop in own code according to there need ?
For example, return an empty buffer ? Or a buffer with a default image or something else  ?

Reply:
Yes. I think this function should be supplied by developer. If not supplied we just crash as it was before.

Reply:
I thought that you call `options.handleImageError` here. Tests would be largerly the same as you wrote initially. 

to test for exception
```
const t = new XlsxTemplate({handleImageError: () => { .... logic ... }});
expect(() => {t.substitute('breaking_image', {
    imgPath : "image_does_not_exist.png",
})}).toThrow(TypeError);

```

Reply:
Similar to other cases. You better rework your last 2 commits, since before that you was prety close.